README.md

```diff
@@ -93,6 +93,14 @@ programming resources.
   `production` one.
 <sup>[[link](#staging-like-prod)]</sup>
 
+* Keep any additional configuration in YAML files under the `config/` directory.
+
+Since Rails 4.2 YAML configuration files can be easily loaded with the new `config_for` method:
```

Comment:
This line and the following snippet are not properly indented with respect to the bullet point.